### ISSUE TITLE
Starting a project with `--at` does not stop the currently running project at that time (`stop_on_start` is `true`)

### ISSUE THREAD

**Issue by tukusejssirs** [TailorDev/Watson#489]
I have set `stop_on_start` to `true`.

When I start a project using `--at` when another project is running, it stops the currently running project at current time, thus it fails to start the new project because it cannot start before the previous project ends.

1. [Current time: `11:00`] `watson start project_1 --at 11:00`
2. [Current time: `12:00`] `watson start project_2 --at 11:30`

```js
// Expected output
Stopping project project_1
Starting project project_2

// Actual output (duration and ID trimmed)
Stopping project project_1
Error: Task cannot start before the previous task ends.
```

**Comment by jmaupetit** [comment]
Yup, I've already experienced this issue, but didn't took time to investigate. Did you?

**Comment by mims-github** [comment]
Experienced the same and think that is a quite common use-case, as one has forgotten to switch project and want to make the switch in the past.
Thx for the helpful tool, like it a lot so far.

**Comment by bobschi** [comment]
i think this was fixed with #480 and can be closed
